services/integrate-android-app-with-custom-service.adoc

```diff
@@ -48,6 +48,23 @@ Now the Aerogear SDK can be used to get the configuration for your custom servic
 
 [source,java]
 ---
-// code example goes here once I can collaborate with Ali
+// get the service URL
```

Comment:
@aliok  Did this example come from an existing code? or is it a standalone example for this doc? 
reason I ask is that we could remote include tagged code if it makes sense in this case (and never have to worry about it becoming stale)

Reply:
@finp From what I can tell, it's just a generic example.